sdk-src/src/main/java/com/hpe/adm/nga/sdk/network/OctaneHttpResponse.java

```diff
@@ -26,13 +32,13 @@
 public class OctaneHttpResponse {
 
     private final int statusCode;
-    private final String content;
     private final InputStream inputStream;
+    private final Charset responseCharset;
 
-    public OctaneHttpResponse(int statusCode, String content, InputStream inputStream) {
+    public OctaneHttpResponse(int statusCode, InputStream inputStream, Charset responseCharset) {
```

Comment:
It was redundant, the response object had both the response body as a string, and as an input stream inside of it. You only really need one of those, The input stream was used for the request binary. But the funny thing is... if you read the content of the http response and parsed it as a string your input stream would be closed. Because on every request, did not matter what kind, you parsed the response as a string (even if it's a 100mb file), the input stream that you also set would never actually work, so to avoid this, I now only set the response input stream into the response. If you want to get the content as a string, inside the response there's a method that will convert the input stream into a string.